e2e/rbd.go

```diff
@@ -590,7 +592,16 @@ var _ = Describe("RBD", func() {
 					e2elog.Failf("failed to validate pvc and application binding with error %v", err)
 				}
 			})
-
+			By("create a Block mode PVC-PVC clone and bind it to an app", func() {
+				v, err := f.ClientSet.Discovery().ServerVersion()
+				if err != nil {
+					e2elog.Failf("failed to get server version with error %v", err)
+				}
+				// pvc clone is only supported from v1.16+
+				if v.Major > "1" || (v.Major == "1" && v.Minor >= "16") {
+					PVCCloneAndValidate(rawPvcPath, pvcBlockSmartClonePath, appBlockSmartClonePath, f)
```

Comment:
I must have missed this earlier, but `rawPvcPath` is badly named too. No need to update it with this PR, unless you want/need to send a new version.

Reply:
Yeah, will put everything in next PR.. thanks